client/NetPacksClient.cpp

```diff
@@ -813,11 +813,21 @@ void ApplyClientNetPackVisitor::visitBattleNextRound(BattleNextRound & pack)
 	callBattleInterfaceIfPresentForBothSides(cl, pack.battleID, &IBattleEventsReceiver::battleNewRound, pack.battleID);
 }
 
+uint64_t timeElapsed(std::chrono::time_point<std::chrono::high_resolution_clock> start)
+{
+	auto end = std::chrono::high_resolution_clock::now();
+
+	return std::chrono::duration_cast<std::chrono::milliseconds>(end - start).count();
+}
+
```

Comment:
Debug code?

If you want to keep it - then probably better to reuse `CStopWatch`. And potentially - upgrade it to use `steady_clock`.

Also - `high_resolution_clock` is problematic since it is not guaranteed to be steady...